migrations/004-add-prlabels-enum-value.js

```diff
@@ -0,0 +1,12 @@
+module.exports.up = function up(queryInterface) {
+  if (queryInterface.sequelize.options.dialect === 'postgres') {
+    // alter enum type
+    return queryInterface.sequelize.query(`ALTER TYPE zappr_data.enum_checks_type ADD VALUE IF NOT EXISTS 'pullrequestlabels';`)
+  }
+  // for sqlite we do not care as it is used for development
+  return Promise.resolve()
+}
+module.exports.down = function down() {
+  // there is no good downgrade
```

Comment:
We could, I guess. But since I found no easy way to safely remove a value from an enum type, I'm not sure what would be gained. You would get a warning you can't do anything about?